security/index.html

```diff
@@ -2,7 +2,16 @@
 layout: default
 title: "Ruby on Rails: Security Policy"
 ---
-    <div id="article" class="security"><article>
+    <div id="article"><article>
+      <header>
+        <h1>Security policy</h1>
+        <h2>
+          Did you find a security vulnerability in Ruby on Rails?<br />
+          Follow these steps to safely report the issue to the core team.<br />
+          You might even be eligible for a bounty.
```

Comment:
I'd remove the bounty program. In the future we may not be doing any bounty. Other than that seems good.